src/main/java/lotto/domain/LottoGamePlayer.java

```diff
@@ -0,0 +1,50 @@
+package lotto.domain;
+
+import lotto.common.ErrorCode;
+import lotto.common.WinningType;
+import lotto.view.ResultView;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class LottoGamePlayer {
+    ResultView resultView = new ResultView();
+
+    public void play(LottoTicket winningLottoTicket, LottoTickets lottoTickets, int bonusNumber) {
+        throwDuplicatedBonusBallException(winningLottoTicket, bonusNumber);
+        WinningType[] winningTypes = lottoTickets.getWinningTypes(winningLottoTicket, bonusNumber);
+        resultView.printResultStatistics(getGameResult(winningTypes));
+    }
+
+    private void throwDuplicatedBonusBallException(LottoTicket winningLottoTicket, int bonusNumber) {
+        if ( winningLottoTicket.contains(new LottoNumber(bonusNumber))) {
```

Comment:
```suggestion
        if (winningLottoTicket.contains(new LottoNumber(bonusNumber))) {
```

그리고 `LottoNumber` 정적 팩토리 `method`가 하나 정도 있으면 좋을 것 같습니다 😄 
그럼 `int`형을 손쉽게 `LottoNumber`로 변경할 수 있으니까요 😄 ex) `LottoNumber.of(bonusNumber)`

Reply:
추가하였습니다.